package.json

```diff
@@ -6,6 +6,7 @@
   "author": "Daniele Bertella <daniele.bertella@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "classnames": "^2.2.6",
```

Comment:
Oh my, I wouldn't use classnames at all if possible, I would stick to sx and them-ui props

Reply:
You can remove this if we decide we are not going to use classes